queries.sql

```diff
@@ -8,3 +8,30 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5 ;
 SELECT * FROM animals WHERE neutered = true ;
 SELECT * FROM animals WHERE name <> 'Gabumon' ;
 SELECT * FROM animals where weight_kg >= 10.4 and weight_kg <= 17.3 ;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+begin;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+COMMIT;
+
```

Comment:
- [ ]  @enocol Great work adding this query, for us to verify the changes that happened before and after the commit we need to add SELECT species from animals before and after the commit to verify the changes happened, kindly check the example in the first query